src/config.ts

```diff
@@ -147,12 +147,11 @@ export interface VLOnlyConfig {
   countTitle?: string;
 
   /**
-   * Whether to filter invalid values (`null` and `NaN`) from the data.
-   * - By default (`undefined`), only quantitative and temporal fields are filtered.
-   * - If set to `true`, all data items with null values are filtered.
-   * - If `false`, all data items are included. In this case, null values will be interpreted as zeroes.
+   * Defines how Vega-Lite should handle invalid values (`null` and `NaN`).
+   * - If set to `"filter"` (default), all data items with null values are filtered.
+   * - If `null`, all data items are included. In this case, invalid values will be interpreted as zeroes.
    */
-  filterInvalid?: boolean;
+  invalidValues?: 'filter';
```

Comment:
> Change config's filterInvalid: boolean to invalidValues: "filter" to support future extension for other ways to handle invalid values. For example, we could have a config where we make invalid values grey (to help distinguish them from zero values) instead of filtering.